### PR TITLE
added recipe for `tiatoolbox`

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
@conda-forge/core there is a bug regarding the linting service.

**Comment by sarthakpati** [comment]
Waiting on https://github.com/TissueImageAnalytics/tiatoolbox/pull/265

**Comment by sarthakpati** [comment]
Hi @conda-forge/python, @conda-forge/staged-recipes, this is ready for review!

File: recipes/tiatoolbox/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  run:
+    - albumentations >0.5.0
+    - click >=7.0
+    - defusedxml
+    - glymur
+    - imagecodecs
+    - matplotlib-base
+    - numpy
+    - opencv >=4.5.4
+    - openslide-python ==1.1.2
```
Comment (jaimergp) [inline recipes/tiatoolbox/meta.yaml:35]: These hard constrains are usually a bad idea. Is this the only version guaranteed to work with this package?

File: recipes/tiatoolbox/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - pandas
+    - pillow
+    - python >=3.7
+    - pyyaml >=5.1
+    - requests
+    - scikit-image ==0.18.3
```
Comment (jaimergp) [inline recipes/tiatoolbox/meta.yaml:41]: Same as above:

> These hard constrains are usually a bad idea. Is this the only version guaranteed to work with this package?

File: recipes/tiatoolbox/meta.yaml (hunk 1 above)
Comment (sarthakpati) [reply recipes/tiatoolbox/meta.yaml:35]: Unfortunately, yes. This is defined in the package itself.

Comment (shaneahmed) [reply recipes/tiatoolbox/meta.yaml:35]: yes this is the latest version of openslide-python. The previous versions break on latest versions of python.

File: recipes/tiatoolbox/meta.yaml (hunk 2 above)
Comment (shaneahmed) [reply recipes/tiatoolbox/meta.yaml:41]: the latest version of scikit-image produces incorrect results. This needs a bit of work before latest version of scikit-image can be used.

File: recipes/tiatoolbox/meta.yaml (hunk 1 above)
Comment (jaimergp) [reply recipes/tiatoolbox/meta.yaml:35]: In that case I'd recommend open upper bounds:

```suggestion
    - openslide-python >=1.1.2
```

Comment (shaneahmed) [reply recipes/tiatoolbox/meta.yaml:35]: that should be fine

File: recipes/tiatoolbox/meta.yaml (hunk 2 above)
Comment (jaimergp) [reply recipes/tiatoolbox/meta.yaml]: Ok, fine, but let's add a comment right above so we can relax the restraint once the bad versions have been identified!

Something like:

```
# TODO: Investigate if later releases of scikit-image work once available
```

Comment (sarthakpati) [reply recipes/tiatoolbox/meta.yaml]: Done.

**Comment by jaimergp** [comment]
Thanks for your submission and patience during the review! Your feedstock will be created soon!

**Pull request merged**